src/Sulu/Component/Content/Tests/Unit/Block/BlockContentTypeTest.php

```diff
@@ -752,7 +765,7 @@ public function testGetContentDataWithEmptySettings()
                         'settings' => new \stdClass(),
                     ],
                 ],
-                'settings' => [],
+                'settings' => new \stdClass(),
```

Comment:
why `stdClass`?

Reply:
Because an array with string keys is used here, which is serialized to a JSON object. And the problem with PHP is, that an empty array is serialized to a JSON array instead of object, leading to some weird inconsistencies.